.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,34 @@
+name: Feature request
+description: "新機能の提案"
+labels: ["enhancement"]
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        [Issueの立て方](https://github.com/TUS-OSK/quantum-gomoku/tree/develop/docs/0.HowToDev/02.DevProcess#issue%E3%81%AE%E7%AB%8B%E3%81%A6%E6%96%B9)を参考に記述してください。
+  - type: textarea
+    attributes:
+      label: "Problem"
+      description: "詳細を記述してください。"
+      placeholder: |
+        以下のようなことを添えて記述してください。
+        - issue number
+          - 例: fixed #123
+        - 問題点(対応内容・対応背景・妥協点)
+        - その他(参考リンクや注意点や補足事項等)
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: "Expected behavior"
+      description: |
+        どのような挙動が期待されるか記述してください。
+      placeholder: |
+        以下のようなことを添えて記述してください。
+        - 期待される挙動
+          - before/after(どのような変更かを画像や動画などで補足するとより良い)
+        - その他(参考リンクや注意点や補足事項等)
```

同様にTDDより、テストを書く指示文が欲しいです
```suggestion
        以下のようなことを添えて記述してください。
        - 期待される挙動
          - before/after(どのような変更かを画像や動画などで補足するとより良い)
          - どういったテストで検証されるか(可能であれば)
```